listeners.go

```diff
@@ -400,16 +453,20 @@ func JoinNetworkAddress(network, host, port string) string {
 	return a
 }
 
+func (na NetworkAddress) ListenQUIC(ctx context.Context, portOffset uint, config net.ListenConfig, tlsConf *tls.Config) (http3.QUICEarlyListener, error) {
```

Comment:
Let's put a godoc comment on this; you could even do something like "LisenQUIC is the same as ListenQUICWithSocket but without a special socket."

Reply:
both funcs have doc comments now 👍